### DISCUSSION TITLE
Customize splash image on bgrt theme when there is a firmware image

### DISCUSSION THREAD

**Discussion by NextbrickInc** [elementary/plymouth-theme#29] [Q&A] [answered]
### Problem

I would like to supply a custom splash image in situations where the device does supply a firmware image.  Running on Ubuntu 22, with servers, a desktop, a laptop and a handful of VMs.

If the device does not have a firmware image, then I can supply my own image for bgrt by placing a png image at `/usr/share/plymouth/themes/spinner/background.png` or `/usr/share/plymouth/themes/spinner/bgrt-fallback.png`.  This works well.

But on systems that DO supply a firmware image, the firmware image is always used, and I have not found a way to override it.  The dell desktop, framework laptop, and all the VMs do supply a firmware image, which bgrt displays faithfully.

### Proposal

Add a way to always use a provided image, or tell me how I can do it with existing functionality.

**Comment by vjr** [comment, marked as answer]
Try set this (and other lines) to false? https://github.com/elementary/plymouth-theme/blob/master/elementary/elementary.plymouth#L27

**Comment by vjr** [comment]
If just that change does not work then see https://github.com/elementary/plymouth-theme/pull/20/files for additional settings and file to add.
